pennylane_lightning/lightning_kokkos/_measurements.py

```diff
@@ -115,3 +115,26 @@ def _process_single_shot(samples):
         return (
             tuple(zip(*processed_samples)) if shots.has_partitioned_shots else processed_samples[0]
         )
+
+    def probs(self, measurementprocess: MeasurementProcess):
+        """Probabilities of the supplied observable or wires contained in the MeasurementProcess.
+
+        Args:
+            measurementprocess (StateMeasurement): measurement to apply to the state
```

Comment:
We might do a pass on all docstring, using either `.` or no `.` in arg and return descriptions.
```suggestion
            measurementprocess (StateMeasurement): measurement to apply to the state.
```